media-sound/snapcast/files/snapclient.initd

```diff
@@ -0,0 +1,18 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+name=snapclient
+command="/usr/bin/snapclient"
+command_args="-d $USER_OPTS $SNAPCLIENT_OPTS"
```

Comment:
I am installing config files from upstream with
`newconfd "${S}/${bin}/debian/snap${bin}.default" "snap${bin}"`
Both vars are set there.

Reply:
I would prefer if you would create an own confd file for Gentoo because the Debian file you are installing will contain things like
```
# start snapclient automatically?
START_SNAPCLIENT=true
```
which won't work in Gentoo but if a user will see something like that he/she will expect that this will work.

And when you create Gentoo's own confd file, take the opportunity to stay in the same variable namespace, e.g. make sure vairables in /etc/conf.d/snapclient will start with `SNAPCLIENT_` if possible.